ui.content/src/main/content/jcr_root/content/dam/hclecomm/en/json/pickup-stores.json/_jcr_content/renditions/original.dir/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:oak="http://jackrabbit.apache.org/oak/ns/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="nt:file">
```

Comment:
can we remove rendition file. I guess it would be atomatically genertaed whe ui.content/src/main/content/jcr_root/content/dam/hclecomm/en/json/pickup-stores.json/.content.xml file will be uploaded